dev-php/symfony-dependency-injection/symfony-dependency-injection-2.1.0.ebuild

```diff
@@ -0,0 +1,35 @@
+# Copyright 1999-2016 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Id$
+
+EAPI=6
+
+DESCRIPTION="Symfony DependencyInjection Component"
+HOMEPAGE="https://github.com/symfony/dependency-injection"
+
+SRC_URI="https://github.com/symfony/dependency-injection/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+
+S="${WORKDIR}/dependency-injection-${PV}"
+
+LICENSE="MIT"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE="test"
+
+RDEPEND="dev-lang/php:*
+	dev-php/fedora-autoloader
+	=dev-php/symfony-yaml-2.1.0
+	=dev-php/symfony-config-2.1.0"
+DEPEND="${RDEPEND}"
+
+src_install() {
+	# I've kept the same path name that Fedora use
+	insinto "/usr/share/php/Symfony/DependencyInjection"
+	doins -r *
+	doins "${FILESDIR}"/autoload.php
+	dodoc README.md
+}
+
+src_test() {
+	phpunit --bootstrap /usr/share/php/Symfony/DependencyInjection/autoload.php || die "test suite failed"
```

Comment:
I am not sure about that one,
Yes it should call the work envirronement source and not the installed one, but we have 2 cases to consider:
* No dependency related, we could change to load the --bootstrap , but that also mean change the autoloader content to be revelant in that context, and so breaking it in the installed context, we could want to duplicate it just for the test.
* When there is multiple libs intended to be loaded the problem become, that we should require them in DEPEND to be installed before, and cause duplicate, but can be done.

Not really sure what choice is better, but the actual situation is the test works *only* if the package is already installed, I aggree this is not good enought, so IMO we could have a test-autoloader for that.